### DISCUSSION TITLE
Getting Call to a member function subscribed() on null

### DISCUSSION THREAD

**Discussion by devhoussam1998** [maartenpaauw/filament-cashier-billing-provider#19] [Q&A]
```
<div>
    @if (auth()->user()->team->subscribed())
        <p class="text-lg text-gray-950 dark:text-white">You are subscribed. 😊</p>
    @endif
</div>
```

**Comment by maartenpaauw** [comment]
I’m sorry. You are opening a lot of discussions lately. And in particular, this one is not even a question and has nothing to do with the plugin. Please do some research before opening issues and provide a good question with actual reproduction steps. I’m gonna close this one.
